src/js/FootprintFetcher.js

```diff
@@ -60,22 +61,104 @@ export async function FetchFootprints(collection, page, step){
       pageInfo += "&limit=" + step;
     }
 
+    // check for pyGeo API
+    if (!collection.url.includes('stac')) 
+    {
+
+        // set offset for 5 & 10 steps
+        offsetMulitiplier = (page * 10 - step);
+        pageInfo = "&offset=" + offsetMulitiplier;
+
+        
+        // checks for 5 change in step
+        if (step <= 10)
+        {
+               
+            // splice limit and change to new limit
+            collectionUrl = collection.url.split('&limit=')[0];
+            collection.url = collectionUrl;
+                
+                
+            // update page pageInfo
+            pageInfo = "&offset=" + offsetMulitiplier + "&limit=" + step;
+            
+            
+        }
+        // checks for 50 & 100 step
+        else if (step == 50 || step == 100)
+        {
+
+            // splice limit and change to new limit
+            collectionUrl = collection.url.split('&limit=')[0];
+            collection.url = collectionUrl;
+
+            // check for first page 
+            if (page == 1)
+            {
+                // set multiplier to 0
+                offsetMulitiplier = 0;
+            }
+            // check for second page
+            else if (page == 2)
+            {   
+                // set multiplier to step
+                offsetMulitiplier = step;
+            
+            }
+            else
+            {
+                // check for 50 and set pages according
+                if (step == 50)
+                {
+                    offsetMulitiplier = page * step - 50;
+                }
+                // check for 100 and set pages according
+                else 
+                {
+                    offsetMulitiplier = page * step - 100;
+                }
+            }
```

Comment:
Can more of this math be done by putting `step` in the equations, as opposed to hardcoded 5, 10, 50, 100?  Maybe some of the ifs could be avoided too?

Reply:
https://github.com/DOI-USGS/asc-geostac/commit/e042fd04969144293fb7147ec9cbb232b53520a4

Reply:
fixed!

Reply:
Now it only seems to load the page starting at item 11. Example: items 11-20 if step is 10, items 11-60 step is 50, etc... and just loads duplicates instead of loading more if "load more" is pressed more than once.

I think it was simplified too much.  I thought there might be a way to simplify from the old code, but it did seem to work, so maybe we should go back to that for the paging math.

Reply:
83ea8eb

Reverted